qiita_db/handlers/prep_template.py

```diff
@@ -63,3 +66,16 @@ def get(self, prep_id):
             response = {'data': pt.to_dataframe().to_dict(orient='index')}
 
         self.write(response)
+
+
+class PrepTemplateAPItestHandler(OauthBaseHandler):
+    @authenticate_oauth
+    def post(self):
+        prep_info_dict = loads(self.get_argument('prep_info'))
```

Comment:
That's a good question - I will open an issue to keep track of this. In this case, this endpoint is tailored only for being able to execute the plugin tests and it will not be exposed in a live environment. Do you agree with this solution?